xml/System.Data.Spatial/DbGeometry.xml

```diff
@@ -1372,8 +1372,8 @@
         <ReturnType>System.Nullable&lt;System.Int32&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the number of points in this DbGeometry value, if it represents a linestring or linear ring. &lt;returns&gt;The number of elements in this geometry value, if it represents a linestring or linear ring; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <value>The number of points in this DbGeometry value.</value>
+        <summary>Gets the number of points in this DbGeometry value, if it represents a linestring or linear ring.</summary>
+        <value>The number of elements in this geometry value, if it represents a linestring or linear ring; otherwise <see langword="null" />.</value>
```

Comment:
```suggestion
        <value>The number of elements in this geometry value, if it represents a linestring or linear ring; otherwise, <see langword="null" />.</value>
```